lua/diagnosticls-configs/linters/swiftformat.lua

```diff
@@ -0,0 +1,30 @@
+local fs = require('diagnosticls-configs.fs')
+
+return {
+  sourceName = 'swiftformat',
+  command = fs.executable('swiftformat'),
+  isStdout = false,
+  isStderr = true,
+  args = {
+    '--lint',
+    'stdin',
+    '--stdinpath',
+    '%filepath'
+  },
+  debounce = 100,
+  offsetLine = 0,
+  offsetColumn = 0,
+  formatLines = 1,
+  formatPattern = {
+    [[.*:(\d+):(\d+): (\w+): (.*)(\r|\n)*$]],
+    { line = 1, column = 2, security = 3, message = { '[swift-format] ', 4 } },
+  },
+  securities = {
+    warning = 'warning'
```

Comment:
For some reason if I don't specify this, it takes 'warning' as an error.

Reply:
I believe that's because if no securities is specified then diagnostic language server will default to any pattern matched found to be an error. This should be fine.

Reply:
Does it output any other words besides "warning" in the security section?

Reply:
Not atm. Just "warning".